jorgeRafaelSJ/day2.html

```diff
@@ -0,0 +1,36 @@
+<!doctype html>
+<html>
+<head>
+	<meta charset = "UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+	<link rel="stylesheet" href="style.css">
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+</head>
+<body>
+	<div class="container">
+		<div class="row">
+			<div class="col-sm-6 questions">
+				<h3>Homework Questions</h3>
+					<ul>
+						<li>When doing a sort() with numbers, I would like to better understand the <em>function(a,b){ return a-b }</em>. So mysterious!</li>
+						<li>Was not sure what was the mission of exercise #10. Only thing I could think of was writing them down manually into arrays and nested arrays.</li>
```

Comment:
@jorgeRafaelSJ see my response to Tash https://github.com/sf-wdi-22-23/23-homework/pull/22 about regexp